src/assets/i18n/cy.json

```diff
@@ -68,7 +68,8 @@
       "forwardParkObservation": "Parcio Ymlaen - gwyliadwriaeth",
       "vehicleChecks": "Gwirio’r cerbyd",
       "controlledStop": "Stopio dan Reolaeth",
-      "eyesightTest": "Prawf Golwg"
+      "eyesightTest": "Prawf Golwg",
+      "uncoupleRecouple": "Uncouple / Adennill"
```

Comment:
Is the welsh for Uncouple  uncouple? or does it just not exist in the welsh language?

Reply:
I am reliably (?) informed it is the same in Welsh and English. Matt tells me all translations will be professionally checked at a later date.